sdk/python/jobs/pipelines/1k_demand_forecasting_with_pipeline_components/automl-forecasting-demand-hierarchical-timeseries-in-pipeline/scripts/helper_scripts.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, Optional, List
+
+import argparse
```

Comment:
Please see the comments for the many model notebook helper script.

Reply:
Fixed.